api/src/queries/survey/survey-occurrence-queries.ts

```diff
@@ -486,3 +486,43 @@ export const getOccurrenceSubmissionMessagesSQL = (occurrenceSubmissionId: numbe
 
   return sqlStatement;
 };
+
+/**
+ * SQL query to get validation schema for a submission based on the occurrence_submission_id.
+ *
+ * @param {number} occurrenceId
+ * @returns {SQLStatement} sql query object
+ */
+export const getValidationSchemaSQL = (occurrenceId: number): SQLStatement | null => {
```

Comment:
can we add simple test for this function? the queries tests are usually easy

Reply:
sure